autoload/airline/extensions.vim

```diff
@@ -138,7 +138,9 @@ function! airline#extensions#load()
     call airline#extensions#netrw#init(s:ext)
   endif
 
-  call airline#extensions#ycm#init(s:ext)
+  if exists(':YcmDiag')
```

Comment:
`g:loaded_youcompleteme` but personally I would advise against it, or at least offer a way to load the extension on demand later for the reason described above.